lib/src/builder.dart

```diff
@@ -98,7 +99,11 @@ class GeneratorBuilder extends Builder {
 
 Stream<GeneratedOutput> _generate(LibraryElement unit,
     List<Generator> generators, BuildStep buildStep) async* {
-  for (var element in getElementsFromLibraryElement(unit)) {
```

Comment:
`severe` logs are fatal in bazel and build_barback.
They are tolerated in build_runner but we want to change that: https://github.com/dart-lang/build/issues/215